packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

```diff
@@ -526,6 +526,14 @@ export default {
             reactHooksMap.set(codePathSegment, reactHooks);
           }
           reactHooks.push(node.callee);
+        } else if (node.arguments.some(isHook)) {
```

Comment:
Note: To prevent passing [hooks as callback to other hooks](https://github.com/facebook/react/issues/22254#issuecomment-913974187) then this should be changed to
```suggestion
        } if (node.arguments.some(isHook)) {
```

Reply:
Please do change it, but fix the formatting.